server/revision/index.js

```diff
@@ -23,5 +24,6 @@ function getRevision(req, _res, next, revisionId) {
 
 module.exports = {
   controller: ctrl,
```

Comment:
Why is controller also exported?

Reply:
@vladimyr wanted to left as it was before.

Reply:
@lukasaric But you removed it from other places?